packages/lab/src/scrim/Scrim.tsx

```diff
@@ -150,60 +51,6 @@ export const Scrim = forwardRef<HTMLDivElement, ScrimProps>(function Scrim(
 
   const scrimRef = useRef<HTMLDivElement>(null);
   const setWrapperRef = useForkRef(ref, scrimRef);
-  const undoAria = useRef(noop);
-  const undoSelection = useRef(noop);
-  const undoTabIndex = useRef(noop);
-  const scrimId = useId();
-
-  useEffect(() => {
-    if (open && !containerRef?.current) {
-      scrims.add(scrimId);
-      noScroll.on();
-    }
-
-    return () => {
-      if (open) {
-        scrims.delete(scrimId);
-        if (scrims.size === 0) {
-          noScroll.off();
-        }
-      }
-    };
-  }, [open, containerRef, scrimId]);
-
-  useEffect(() => {
-    if (open) {
-      const parent = containerRef?.current || undefined;
-
-      if (scrimRef.current) {
-        undoAria.current = hideOthers(scrimRef.current, parent);
-      }
-    }
-
-    return () => {
-      undoAria.current?.();
-    };
-  }, [open, containerRef]);
-
-  useEffect(() => {
-    if (open) {
-      const parent = containerRef?.current;
-
-      if (parent) {
-        undoSelection.current = preventSelection(parent);
-        undoTabIndex.current = preventFocusOthers(
-          scrimRef.current,
-          tabEnabledSelectors,
-          parent
-        );
-      }
-    }
-
-    return () => {
-      undoSelection.current?.();
-      undoTabIndex.current?.();
-    };
-  }, [open, containerRef, tabEnabledSelectors]);
 
   useEffect(() => {
     if (closeWithEscape && open && scrimRef.current) {
```

Comment:
The logic here can be simplified, instead of use ownerDocument we can use the targetWindow where the style is injected